### DISCUSSION TITLE
Ideal or common ways of adding custom charts to EKS cluster

### DISCUSSION THREAD

**Discussion by gitsstewart** [gruntwork-io/knowledge-base#573] [Help] [answered]
Customer Question:
Do we have any methods or way to add any custom chart to eks cluster like rabbitmq or any other helm chart we want to add? 
There are lots of plugins available in the market that in future we might want to add, are there any recommended or commonly used? 
Is there a "best" way or tool to add charts to EKS cluster or yaml file that we can apply so its always part of the EKS cluster and manages as IaC?

---

<ins datetime="2022-10-20T19:36:45Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109405">Tracked in ticket #109405</a></p>
</ins>

**Comment by josh-padnick** [comment, marked as answer]
Re-posted in https://github.com/gruntwork-io/knowledge-base/discussions/576.
